content/collections/experiment-results/en/experiment-results-use-formula-metrics.md

```diff
@@ -123,6 +123,18 @@ This function only works when grouping by a numerical property on the last event
 
 Returns the average value of the given property among all the users who completed the conversion. This function internally uses Funnel Query to get the average value among the converted users.
 
+### PROPCOUNT
+
+**Syntax:** `PROPCOUNT(event)`
+
+* **Event:** Refers to the event that interests you. This must be a letter that corresponds to an event in the Events card. If grouping by multiple properties, the formula runs the calculation with the first group by clause.
+
+Returns the number of distinct property values for the property the event is grouped by. In this setup, the formula retrieves the number of different departments covering all the items for which details were viewed:
+
+![propcount_sidecontrols.png](/docs/output/img/event-segmentation/propcount-sidecontrols-png.png)
```

Comment:
this is an image from an event seg chart which is a little weird to have that in an article that talks about experiment results chart